testing_and_setup/compass/ocean/global_ocean/config_files/config_base_mesh.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <config case="base_mesh">
+	<get_file dest_path="work_case_dir" file_name="topo.msh">
```

Comment:
@xylar, I don't think `topo.msh` is actually being used in the workflow right now. It might be a good idea to keep it around for visualization purposes. Was that the intention?

Reply:
i couldn't tell why this was part of so many test cases...